ci-operator/step-registry/hypershift/kubevirt/create/hypershift-kubevirt-create-commands.sh

```diff
@@ -159,10 +159,14 @@ EOF
   fi
 fi
 
+if [[ -f "${SHARED_DIR}/GPU_DEVICE_NAME" ]]; then
+  EXTRA_ARGS="${EXTRA_ARGS} --annotations hypershift.openshift.io/kubevirt-vm-jsonpatch='[{\"op\":\"add\",\"path\":\"/spec/template/spec/domain/devices/gpus/-\",\"value\":{\"deviceName\":\"$(cat "${SHARED_DIR}/GPU_DEVICE_NAME")\",\"name\":\"gpu1\"}}]'"
```

Comment:
Please use the `--host-device-name` argument available in the `hcp create cluster kubevirt` command, instead of using the jsonpatch annotation to the NodePool.
This should be a supported feature and we have an API for it.